charts/cluster-scanner/templates/rbac.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "cluster-scanner.fullname" . }}
+rules:
+- apiGroups: ["", "apps", "batch", "extensions"]
+  resources:
+    - "namespaces"
+    - "deployments"
+    - "replicasets"
+    - "daemonsets"
+    - "statefulsets"
+    - "pods"
+    - "cronjobs"
+    - "jobs"
+    - "nodes"
+    - "secrets"
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["", "coordination.k8s.io"]
+  resources:
+    - "leases"
```

Comment:
Since the lease is going to be used in the bootstrap and is a namespaced resource we should avoid to create it as ClusterRole level and limit to the namespace we are going to use with a Role + RoleBinding (see [#1041](https://github.com/sysdiglabs/charts/pull/1041))